roles/format-golang-test-results/tasks/main.yml

```diff
@@ -4,11 +4,11 @@
       set -e
       set -x
 
-      go get -u github.com/theopenlab/go2xunit
+      go get -u github.com/tebeka/go2xunit
       go2xunit -input $TEST_RESULTS_TXT -output $TEST_RESULTS_XML || true
 
       apt-get install --yes xsltproc
-      wget https://raw.githubusercontent.com/theopenlab/nosetest-xunit-xslt/master/nosetests.xslt
```

Comment:
maybe don't need to wget ? seems there is built-in ansible vars can be used

Reply:
Thank you, will use `get_url`